packages/opentelemetry-core/src/context/propagation/HttpTraceContext.ts

```diff
@@ -30,12 +30,13 @@ export const TRACE_PARENT_HEADER = 'traceparent';
 export const TRACE_STATE_HEADER = 'tracestate';
 
 const VERSION = '00';
-const VERSION_PART_COUNT = 4; // Version 00 only allows the specific 4 fields.
-
-const VERSION_REGEX = /^(?!ff)[\da-f]{2}$/;
-const TRACE_ID_REGEX = /^(?![0]{32})[\da-f]{32}$/;
-const PARENT_ID_REGEX = /^(?![0]{16})[\da-f]{16}$/;
-const FLAGS_REGEX = /^[\da-f]{2}$/;
+const VERSION_REGEX = '(?!ff)[\\da-f]{2}';
```

Comment:
As these are now just strings, it doesn't make sense to keep the variable name with the _REGEX postfix, it's also a little error prone as someone coming alone in the future might either think these are real regex's and/or forget to escape the '\'.

Suggestion:
Either use a different postfix for the names "_PART" "_STRING" etc, or as they are not used elsewhere, just include inline. And include a comment for the components (as named capture groups are not supported by all environments)

Reply:
inlining them doesn't make it more readable. I'll add the `_PART`.